parsers/cxx/cxx_parser.c

```diff
@@ -973,7 +973,7 @@ static bool cxxParserParseClassStructOrUnionInternal(
 			(!(uInitialKeywordState & CXXParserKeywordStateSeenTypedef))
 		)
 	{
-		CXX_DEBUG_PRINT("Maybe KnR funciton definition?");
+		CXX_DEBUG_PRINT("Maybe KnR function definition?");
```

Comment:
Maybe `K&R` is better. However, the string KnR is used in a name of cunction.  I think both words are acceptable.